lib/testing_api_onlyoffice_com/test_instance/main_page/community_server_api.rb

```diff
@@ -1,200 +1,67 @@
-    link(:mailserver, xpath: '//*[contains(@href, "mailserver")]')
-    link(:mailserver_addressdata, xpath: '//*[contains(@href, "mailserver/addressdata")]')
-    link(:mailserver_dnsrecords, xpath: '//*[contains(@href, "mailserver/dnsrecords")]')
-    link(:mailserver_domains, xpath: '//*[contains(@href, "mailserver/domains")]')
-    link(:mailserver_mailboxes, xpath: '//*[contains(@href, "mailserver/mailboxes")]')
-    link(:mailserver_mailgroup, xpath: '//*[contains(@href, "mailserver/mailgroup")]')
-    link(:mailserver_notifications, xpath: '//*[contains(@href, "mailserver/notifications")]')
-    link(:mailserver_servers, xpath: '//*[contains(@href, "mailserver/servers")]')
-    link(:people, xpath: '//*[contains(@href, "people")]')
-    link(:people_reassign_user_data, xpath: '//*[contains(@href, "people/reassign%20user%20data")]')
-    link(:people_remove_user_data, xpath: '//*[contains(@href, "people/remove%20user%20data")]')
-    link(:portal, xpath: '//*[contains(@href, "portal")]')
-    link(:portal_backup, xpath: '//*[contains(@href, "portal/backup")]')
-    link(:portal_quota, xpath: '//*[contains(@href, "portal/quota")]')
-    link(:portal_users, xpath: '//*[contains(@href, "portal/users")]')
-    link(:project, xpath: '//*[contains(@href, "project")]')
-    link(:project_comments, xpath: '//*[contains(@href, "project/comments")]')
-    link(:project_contacts, xpath: '//*[contains(@href, "project/contacts")]')
-    link(:project_discussions, xpath: '//*[contains(@href, "project/discussions")]')
-    link(:project_files, xpath: '//*[contains(@href, "project/files")]')
-    link(:project_import, xpath: '//*[contains(@href, "project/import")]')
-    link(:project_milestones, xpath: '//*[contains(@href, "project/milestones")]')
-    link(:project_projects, xpath: '//*[contains(@href, "project/projects")]')
-    link(:project_report, xpath: '//*[contains(@href, "project/report")]')
-    link(:project_tags, xpath: '//*[contains(@href, "project/tags")]')
-    link(:project_tasks, xpath: '//*[contains(@href, "project/tasks")]')
-    link(:project_team, xpath: '//*[contains(@href, "project/team")]')
-    link(:project_template, xpath: '//*[contains(@href, "project/template")]')
-    link(:project_time, xpath: '//*[contains(@href, "project/time")]')
-    link(:settings, xpath: '//*[contains(@href, "settings")]')
-
-    # controls
-    elements(:expandable_hitarea, xpath: '//*[contains(@class, "hitarea expandable-hitarea")]')
-
     def initialize(instance)
       @instance = instance
       super(@instance.webdriver.driver)
       wait_to_load
+      @documentation_objects = init_navigation_objects
     end
 
     def wait_to_load
       @instance.webdriver.wait_until { identification_element.visible? }
     end
 
-    def expand_every_expandable_area
-      return if expandable_hitarea_elements.size.zero?
-
-      expandable_hitarea_elements.each do |element|
-        element.click
-        sleep 1
+    def init_navigation_objects
```

Comment:
There is almost same method in `BuilderPage`, perhaps there is a way to make one universal methods